scripts/rebuildstaging

```diff
@@ -47,7 +47,9 @@ done
 
 function rebuildstaging() {
     echo "rebuilding staging branch, this might take a while..."
-    git-build-branch scripts/staging.yml "$@"
+    curl --silent https://raw.githubusercontent.com/dimagi/staging-branches/main/commcare-hq-staging.yml > scripts/tmp-staging.yml
+    git-build-branch scripts/tmp-staging.yml "$@"
+    rm scripts/tmp-staging.yml
```

Comment:
nit: Should `scripts/tmp-staging.yml` (or rather `/scripts/tmp-staging.yml`) be added to .gitignore? I think this line might not run, depending on bash flags, if `git-build-branch` fails.

Reply:
So I decided to save the file to `artifacts/staging.yml`. This directory is ignored already, and by saving it to a directory that is ignored, we don't have to worry about cleaning it up which means a user can view it to debug any issues if needed. Artifacts also felt appropriate enough for this, but open to thoughts. 358ae59f9bfe552a2c73327231e2c67f41808a64